src/components/tooltip/tooltip--icon.hbs

```diff
@@ -1,10 +1,7 @@
 <div class="bx--tooltip--icon">
-  <div tabindex="0" class="bx--tooltip__trigger bx--tooltip--icon__bottom" data-tooltip-text="Filter">
+  <button class="bx--tooltip__trigger bx--tooltip--icon__top" title="Filter">
```

Comment:
I think we could run into an issue with IE11 screenreaders since they have trouble supporting CSS-generated content (e.g. `content: attr(title)`) 

Reference: https://tink.uk/accessibility-support-for-css-generated-content/

Reply:
I thought the screen reader would just read the title attribute? Thats why I switched it to title instead of the data attribute. I could be wrong though

Reply:
Oh duh, that totally works 💯

Reply:
@alisonjoseph will the browser tooltip also show up when we hover if we use the `title` attribute btw?